### ISSUE TITLE
A11y Report: Resolving Major Accessibility Issues

### ISSUE THREAD

**Issue by wintere** [LondonStageDB/website#90]
Please read the attached [report-results.html](https://github.com/user-attachments/files/24263682/report-results.html) generated by the @uo-zz's accessibility pipeline [repository](https://github.com/LondonStageDB/a11y-auto-testing) first. This is not an exhaustive list, but it does cover the errors that affect multiple pages or files.

## Reformat Footer Links on LSDB for Accessibility

Some links on the LSDB website rely on color alone to distinguish between textual content and hyperlinks. To adhere to WCAG 2.0 guidelines, we can either increase the contrast in color between the background and link text, or [add additional styling (like italics)](https://testingbot.com/support/accessibility/web/rules/link-in-text-block) to better distinguish links from text.

### Affected Files
* common/footer.php 
  - This means that almost every page on the website is flagged with this error. 
* css/main.css
   - Fixing the uo-footer__link class in main.css should resolve this.

### How to Fix
* Upping the contrast and adding styling should be straightforward enough. Let's find a compromise that passes the test and looks good!
* This could be a good starter project for student workers to ease into editing website content, as the fix involves slightly changing a few lines of CSS on the LSDB website without restructuring content or writing PHP.

## Remove Instances of tabindex > 0

This error is triggered when tabindex is set to any value > 0. Values like 0 and -1 are acceptable.

### Affected Files
* search.php
* sphinx-results.php

### How to Fix
* This guide suggest never setting [tabindex > 0](https://testingbot.com/support/accessibility/web/rules/tabindex#what-is-being-tested) whenever possible.

## Ensure All Page Content is Contained by Landmarks

Some pages have some content that isn't contained within proper HTML5 landmarks.

#### No Main Error
Either `<div role="main">` or `<main>` is acceptable. Apparently, `class="main"` and `id="main"` aren't.
  - index.php
  - search.php
  - data.php
  - about-data.php
  - glossary.php
  - cast-list.php
  - authors.php
  - dates.php
  - tips.php
  - about.php
  - history.php
  - team.php
  - citation.php
  - media-coverage.php
  - contact.php
  - news.php
  - sphinx-results.php
  - event.php

### How to Fix
* Make sure all [page content is contained ](https://testingbot.com/support/accessibility/web/rules/region#supportContainer) within landmark regions like `<header><main>` and `<nav>`.
